opm/polymer/PolymerProperties.hpp

```diff
@@ -147,14 +147,14 @@ namespace Opm
             const auto& plyviscTable = tables->getPlyviscTables().getTable<PlyviscTable>(0);
 
 
-            c_vals_visc_ = plyviscTable.getPolymerConcentrationColumn();
-            visc_mult_vals_ =  plyviscTable.getViscosityMultiplierColumn();
+            c_vals_visc_ = plyviscTable.getPolymerConcentrationColumn().vectorCopy( );
```

Comment:
Change of opinion; currently (I think) the state of the matter is that `exportColumn(  )` is used in opm-core and `vectorCopy( )` is used in opm-autodiff. As mentioned in the opm-core PR I would like to go to JSON based support of the column names. I have not been able to determine for myself which of the two is the best intermediate stage.

I would happy to unify though.